### PR TITLE
[14.0][FIX] account _invoice_line_sequence : use a flag on company to avoid breaking tests

### PR DESCRIPTION
**Pull request by jguenat**
- false by default to avoid breaking tests of other modules #1382

### CODE REVIEW

**Review by TDu — APPROVED** [review]
LG, thank you for doing that.

File: account_invoice_line_sequence/__manifest__.py [hunk 1]
```diff
@@ -5,15 +5,19 @@
 
 {
     "name": "Account Invoice line with sequence number",
-    "version": "14.0.1.0.1",
+    "version": "14.0.2.0.1",
```
Comment (jbaudoux) [inline account_invoice_line_sequence/__manifest__.py:8]: The migration script should not be in a directory including the last digit ?
"14.0.2.0.1" vs "14.0.2.0"

Comment (jguenat) [reply account_invoice_line_sequence/__manifest__.py:8]: indeed, corrected
It was working but it's nicer like that

**Review by jbaudoux — APPROVED** [review]

**Review by mt-software-de — APPROVED** [review]
LGTM

**Comment by bosd** [comment]
Why Introduce a new setting while the goal is to fix test compatibility?

Why not use a conditional context on the default test value?

**Comment by jguenat** [comment]
> Why Introduce a new setting while the goal is to fix test compatibility?
> 
> Why not use a conditional context on the default test value?

it's discussed here https://github.com/OCA/account-invoicing/pull/1382#issuecomment-1455664425

**Comment by bosd** [comment]
> it's discussed here [#1382 (comment)](https://github.com/OCA/account-invoicing/pull/1382#issuecomment-1455664425)

Thanks fro the clarification.

**Review by AaronHForgeFlow — APPROVED** [review]
Code LG

**Comment by hbrunn** [comment]
ocabot is merging for quite a while already, so I take the opportunity to ask: Did you try to just not write the sequence field? I've disentangled sequence and sequence2 in the v15 (#1565) migration, and I think this is actually a better fix

**Comment by AaronHForgeFlow** [comment]
The merge is aborted. Better to backport the refactor from https://github.com/OCA/account-invoicing/pull/1565 at this point. Otherwise 13.0, 14.0 and 15.0 would have very different code.

**Comment by dreispt** [comment]
Superseded by https://github.com/OCA/account-invoicing/pull/1565

**Pull request closed without merging**